src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/floodlight/model/FlowSegmentMetadata.java

```diff
@@ -38,10 +38,9 @@ public class FlowSegmentMetadata implements Serializable {
     @JsonCreator
     public FlowSegmentMetadata(
             @JsonProperty("flowid") @NonNull String flowId,
-            @JsonProperty("cookie") @NonNull Cookie cookie,
-            @JsonProperty("multi_table") boolean multiTable) {
+            @JsonProperty("cookie") @NonNull Cookie cookie) {
         this.flowId = flowId;
         this.cookie = cookie;
-        this.multiTable = multiTable;
+        this.multiTable = true;
```

Comment:
Is it really needed? It can only have one value, right?

Reply:
yes, it will have only one value. I will have 2 more PRs (e.g #5382) which will remove fields from Java objects